pkg/bundle/source.go

```diff
@@ -99,27 +99,20 @@ func (b *bundle) buildSourceBundle(ctx context.Context, bundle *trustapi.Bundle)
 			return bundleData{}, fmt.Errorf("failed to retrieve bundle from source: %w", err)
 		}
 
-		opts := util.ValidateAndSanitizeOptions{FilterExpired: b.Options.FilterExpiredCerts}
-		sanitizedBundle, err := util.ValidateAndSanitizePEMBundleWithOptions([]byte(sourceData), opts)
+		err = util.ValidateAndSanitizePEMBundleWithOptions(certPool, []byte(sourceData))
```

Comment:
Since we plan to break an exported function, could we cleanup a bit in the util package? I think it would be better to use `ValidateAndSplitPEMBundle` here, since the options are now "hidden" inside the cert pool.